src/tools/pbs_upgrade_job.c

```diff
@@ -41,10 +41,11 @@
  *
  * @brief
  *		pbs_upgrade_job.c - This file contains the functions to read in
- *		an older .JB file (pre 13.0) and convert it into the newer format.
+ *		an older .JB file (pre 19.0) and convert it into the newer format.
  * @par
- *		This tool is required due to the change in PBS_MAXSERVERNAME that
- *		altered the size of the jobfix and taskfix structures.
+ *		This tool is required due to the change in PBS macros which are defined in the pbs_ifl.h/server_limits.h
+ 		or in any other file and the same macros PBS uses in the job structure(see job.h) as well that alters the
+		size of the jobfix and taskfix structures.
```

Comment:
because previous description was claiming that tool is required only for change in single macro (PBS_MAXSERVERNAME), but instead of that PBS uses other macros as well in job structure.